src/components/AuditLogsTable/AuditLogsTable.tsx

```diff
@@ -66,14 +71,16 @@ const AuditLogsTable = ({ showModel = false }: Props) => {
     if (!wsControllerURL || !hasControllerConnection) {
       return;
     }
-    setLoading(true);
-    promiseDispatch<AuditEvents>(
-      jujuActions.findAuditEvents({ wsControllerURL })
-    ).then((response) => {
-      setLoading(false);
-      setAuditLogs(response?.events);
-    });
-  }, [hasControllerConnection, promiseDispatch, wsControllerURL]);
+    dispatch(jujuActions.fetchAuditEvents({ wsControllerURL }));
```

Comment:
Dispatching this action will fetch the audit events in the middleware as before, but now the audit events end up in the Redux store instead of being returned directly to this component.